client/payment-details/summary/index.tsx

```diff
@@ -191,6 +368,9 @@ const PaymentDetailsSummary: React.FC< PaymentDetailsSummaryProps > = ( {
 		},
 	} );
 
+	const resolvedDisputeMessage =
+		charge.dispute && composeResolvedDisputeMessage( charge.dispute );
```

Comment:
Appending content like this implies knowledge of the surrounding and contained markup – are all these components (incl. `composeResolvedDisputeMessage`) tightly coupled?

Reply:
Refactored to separate components in d532d23.